forum/users/models.py

```diff
@@ -1,3 +1,26 @@
+from django.core.validators import EmailValidator
 from django.db import models
+import uuid
 
 # Create your models here.
+class Role(models.Model):
+    role_id = models.UUIDField(primary_key=True, default=uuid.uuid4)
```

Comment:
Consider adding `editable=False` to ensure that users don’t attempt to change the UUID.
```
role_id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
```

Reply:
Added editable=False for user_id field.